app.rb

```diff
@@ -0,0 +1,10 @@
+letters = [
+  "A: alpha", "B: bravo", "C: charlie", "D: delta", "E: echo", 
+  "F: foxtrot", "G: golf", "H: hotel", "I: india", "J: juliet", 
+  "K: kilo", "lima", "mike", "november", "oscar", 
+  "papa", "quebec", "romeo", "sierra", "tango",
+  "umbrella", "victor", "whiskey", "x-ray", "yankee", 
+  "zulu", "?????", "And, per-se, and", "!!!!!!!", "whatever!", "adding garbage!!!!!",  "So Annoying", 
+ ]
```

Comment:
Style/IndentArray: Indent the right bracket the same as the start of the line where the left bracket is.